src/components/dashboard/Hub/index.tsx

```diff
@@ -140,6 +140,9 @@ function HubRouter() {
     }
   }
 
+  push({
+    search: showWelcome ? '?welcomescreen=true' : 'welcomescreen=false',
```

Comment:
Push a welcomescreen search param to the URL for Google Analytics (based on the showWelcome variable).